packages/zipkin/src/zipkinCore.thrift

```diff
@@ -1,46 +1,448 @@
 # Generate with the command "thrift -gen js:node zipkinCore.thrift"
 namespace java com.twitter.zipkin.gen
-namespace rb Zipkin
 
-#************** Collection related structs **************
-
-# these are the annotations we always expect to find in a span
+#************** Annotation.value **************
```

Comment:
all this here is the result of bumping to the latest thrift (which has comments and a few new things)